src/objclass/class_api.cc

```diff
@@ -112,24 +112,24 @@ int cls_gen_random_bytes(char *buf, int size)
 
 int cls_gen_rand_base64(char *dest, int size) /* size should be the required string size + 1 */
 {
-  char buf[size];
-  char tmp_dest[size + 4]; /* so that there's space for the extra '=' characters, and some */
+  std::vector<char> buf(size);
+  std::vector<char> tmp_dest(size + 4); /* so that there's space for the extra '=' characters, and some */
```

Comment:
It seems odd that the dest-ptr would give us a size that we're going to ignore, though..?